.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: node_js
+
+node_js:
+  - "10.15.3"
+
+services:
+  - postgresql
+
+addons:
+  postgresql: "11.6"
+
+before_script:
+  - npm install
+  - psql -c 'create database testdb;' -U postgres
+script:
+  - npm run test
```

Comment:
I added this because I assumed we had Travis CI applied to this repo. We don't. We could:

1. Add it
  * Pros: integration testing
  * Cons: added complication trying to get it to work, which is probably different from local configurations
2. Remove this file
  * Pros: the file is unused without Travis CI, avoid the complication
  * Cons: no integration testing, only local testing